app/Exports/XlsformTemplateLanguageExport.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+namespace App\Exports;
+
+use App\Models\SurveyRow;
+use App\Models\LanguageString;
+use App\Models\LanguageStringType;
+use App\Models\XlsformTemplateLanguage;
+use PhpOffice\PhpSpreadsheet\Style\Fill;
+use Maatwebsite\Excel\Concerns\FromArray;
+use Maatwebsite\Excel\Concerns\WithTitle;
+use Maatwebsite\Excel\Concerns\WithStyles;
+use Maatwebsite\Excel\Concerns\WithHeadings;
+use PhpOffice\PhpSpreadsheet\Cell\Coordinate;
+use PhpOffice\PhpSpreadsheet\Style\Alignment;
+use Maatwebsite\Excel\Concerns\WithColumnWidths;
+use PhpOffice\PhpSpreadsheet\Worksheet\Worksheet;
+use Maatwebsite\Excel\Concerns\WithBackgroundColor;
+
+class XlsformTemplateLanguageExport implements FromArray, WithHeadings, WithTitle, WithColumnWidths, WithStyles, WithBackgroundColor
+{
+    private $xlsformTemplateId;
+
+    public function __construct($templateId)
+    {
+        $this->xlsformTemplateId = $templateId;
+    }
```

Comment:
This would suggest a slightly different order for creating XlsformTemplateLanguage entries. Perhaps rather than creating a new entry when you upload a completed form translation, you instead create new entries at step 1. This implies:

- XlsformTemplateLanguage entries also belong to a team (the team that does the translation)
- we need a way of showing if a translation has been uploaded for the entry or not (so the team can see which ones they have completed. 

In the example above, maybe when the team picks Spanish, they might decide to use the Spanish Mexico version with no changes. In that case, we still need a way of showing that the team is using that language, even though its not "theirs" (they didn't upload it).